docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -327,11 +326,9 @@ typedef struct _oe_sgx_endorsements_t
      *      crl[0] = CRL for the SGX PCK Certificate
      *      crl[1] = CRL for the SGX PCK Processor CA
      *
-     *  OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_PCK_CERT to
-     *     OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_PCK_PROC_CA
+     *  OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_PCK_CERT
      *  CRLs issuer chains in PEM format, null-terminated string
-     *      crl[0] = Issuer Chain for the SGX PCK Certificate
-     *      crl[1] = CRL for the SGX PCK Processor CA
+     *      Issuer Chain for the SGX PCK Certificate
```

Comment:
The Issuer Chain for the SGX PCK Certificate **CRL**. Actually this info has been described by the sentence above, you might just remove it.

Reply:
Thanks. Updated that.